tests/test_unbound.py

```
@@ -138,6 +328,484 @@ async def test_enable_disable_unbound_blocklist(
+    """
+    client, _session = make_mock_session_client(make_client)
+    try:
+        client.get_host_firmware_version = AsyncMock(return_value="25.7.7")
+        client._safe_dict_get = AsyncMock(
+            return_value={
+                "unbound": {
+                    "dnsbl": {
+                        "enabled": "0",
+                        "safesearch": "0",
+                        "nxdomain": "1",
+                        "address": "0.0.0.0",
```

Ruff (0.15.9)

[error] Possible binding to all interfaces

(S104)